client/app/scripts/superdesk-authoring/tests/authoring_spec.js

```diff
@@ -309,6 +309,45 @@ describe('authoring', function() {
     });
 });
 
+describe('cropImage', function() {
+    beforeEach(module('superdesk.authoring'));
+    beforeEach(module('superdesk.mocks'));
+    beforeEach(module('templates'));
+
+    it('can save crop', inject(function($q, $rootScope, urls, $http, $httpBackend, cropImage) {
```

Comment:
IMO rename `'can <verb>'` ---> `'<verb>s ...'`, because we are not testing that a function _can do_ something, we test that it actually _does_ it. ;)